FlashCap.Core/Devices/DirectShowDevice.cs

```diff
@@ -174,13 +175,24 @@ private sealed class SampleGrabberSink :
 
                     ///////////////////////////////
 
-                    var captureGraphBuilder = NativeMethods_DirectShow.CreateCaptureGraphBuilder();
+                    captureGraphBuilder = NativeMethods_DirectShow.CreateCaptureGraphBuilder();
                     if (captureGraphBuilder.SetFiltergraph(this.graphBuilder) < 0)
                     {
                         throw new ArgumentException(
                             $"FlashCap: Couldn't set graph builder: DevicePath={devicePath}");
                     }
 
+                    ///////////////////////////////
+                    
+                    captureGraphBuilder.FindInterface(Guid.Empty, Guid.Empty, captureSource, NativeMethods_DirectShow.IAMVideoProcAmpHelper.GUID, out object? videoProcAmpObject);
```

Comment:
I can't use `typeof(IAMVideoProcAmp).GUID` because of .netstandard 1.3

any alternative to `NativeMethods_DirectShow.IAMVideoProcAmpHelper.GUID` ?

Reply:
@ldlac There is no elegant way to do this, but it can be redefined in the same way as `IID_IBaseFilter`.